### ISSUE TITLE
Communication Checkboxes are ambiguous

### ISSUE THREAD

**Issue by ghostsquad** [javajuice1337/RustPlusBot#2]
```
[ ] Allow @everyone/@role/@user mentions in team chat messages posted to the Discord channel
[ ] Permanently silence (mute) the bot so it only outputs to the Discord channel
[ ] Post all incoming team chat messages to the Discord channel
[ ] Post only incoming team chat messages mentioning team members to the Discord channel
[ ] Post only team chat bot responses to the Discord channel
[ ] Send messages from the Discord channel to team chat 
```

```
[ ] Post vending machine notifications to the Discord channel
[ ] Send new vending machine notifications to team chat
[ ] Show the new vending machine name in the team chat notification
```

Some of these options seem ambiguous or contradictory.

### `Permanently silence (mute) the bot so it only outputs to the Discord channel`

E.g. if `Permanently silence (mute) the bot so it only outputs to the Discord channel`. Does this mean it only posts to the "main" discord channel, and not to the `Notifications Channel` or `Trackers` channel? Does it further mean that I won't get notifications/tracker messages?

If my intuition on this is correct, I think I would prefer seeing something like the following options:

```
## Discord Messages

[ ] Enable Rust Team Chat Messages (to the Main Discord Channel)
[ ] Enable Notification Messages
[ ] Enable Tracker Messages
```

Further, I'd put the channel selection and these checkboxes next to each other (instead of on separate pages).

### `Post all incoming team chat messages to the Discord channel`

I think this would be equivalent to the above `Enable Rust Team Chat Messages (to the Main Discord Channel)`.

### `Post only incoming team chat messages mentioning team members to the Discord channel`

I'm assuming this is a filter, such that the player's in-game name needs to be included in the message to be forwarded to Discord. If so, maybe it could be reworded to:

```
Post only Rust team chat messages mentioning team members to the Main Discord Channel
```

### `Post only team chat bot responses to the Discord channel`

I have no idea what this means!

### `Send messages from the Discord channel to team chat`

I think this could be reworded as `Send messages from the Main Discord channel to Rust team chat`.

### TLDR

The TLDR is that, it think it would help to disambiguate "the Discord channel" to a specifically labelled channel (main, notifications, trackers), and change `team chat` to `Rust team chat`.

Thank you for your time!

**Comment by javajuice1337** [comment]
Thanks for the detailed suggestions! I've reworked the **RustPlusBot** configuration page and added descriptive tool-tips.

**Comment by ghostsquad** [comment]
❤️ 

![image](https://github.com/javajuice1337/RustPlusBot/assets/903488/316a7f8a-4c27-4ff4-9bb8-46599fa67aa2)

Radio checkboxes is a huge improvement here. That clearly shows that these are mutually exclusive options.
